src/components/DetailsMetrics/detailsMetrics.utils.js

```diff
@@ -229,7 +269,8 @@ export const parseMetrics = (data, timeUnit) => {
       title: getMetricTitle(full_name),
       driftStatusList,
       totalDriftStatus,
-      [withInvocationRate ? 'total' : 'avg']: totalOrAvg
+      [withInvocationRate ? 'total' : 'avg']: totalOrAvg.formattedResult,
+      [withInvocationRate ? 'rawDataTotal' : 'avgRaw']: totalOrAvg.rawResult
```

Comment:
total, avg, rawDataTotal, avgRaw - must be constants

Reply:
done.